sapl/templates/painel/index.html

```diff
@@ -490,6 +508,14 @@ <h2 class="text-subtitle">Tema da Sessão Solene</h2>
                 `<font color="`+color+`">${parlamentar.partido}</font> </td> <td style="padding-right:20px">` +
                 `<font color="`+color+`">${parlamentar.voto}</font> </td></tr>`)
      }
+
+      function changeFontSize(elementId, increment) {
+          let element = document.getElementById(elementId);
+          if (element) {
+              let currentSize = parseFloat(window.getComputedStyle(element, null).getPropertyValue('font-size'));
+              element.style.fontSize = (currentSize + increment) + 'px';
+          }
```

Comment:
Dúvida genuína: teria como usar jQuery ao invés de JavaScript "puro"? Se a resposta for *não*, tudo bem. A gente faz o merge como está. :)